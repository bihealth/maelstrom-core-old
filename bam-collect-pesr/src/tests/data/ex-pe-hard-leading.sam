../../../../bam-scan/src/tests/data/ex-pe-hard-leading.sam