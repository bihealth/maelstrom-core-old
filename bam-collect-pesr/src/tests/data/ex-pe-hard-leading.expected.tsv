#contig	start	end	signal
one	99	149	{"SplitRead":{"read_name":"pair_expected","is_first":false,"contig":"one","start":99,"end":149,"clipped_sides":"Right"}}
one	99	149	{"PairedRead":{"read_name":"pair_expected","is_first1":false,"contig1":"one","start1":99,"end1":149,"strand1":"Forward","contig2":null,"start2":null,"strand2":null,"tlen":null}}
one	100099	100149	{"SplitRead":{"read_name":"pair_expected","is_first":true,"contig":"one","start":100099,"end":100149,"clipped_sides":"Left"}}
